pyincore_data/censusutil.py

```diff
@@ -178,6 +190,7 @@ def get_fips_by_state(state: str, year: str = 2010):
     def get_blockgroupdata_for_dislocation(state_counties: list, vintage: str = "2010", dataset_name: str = 'dec/sf1',
                                            out_csv: bool = False, out_shapefile: bool = False,
                                            out_geopackage: bool = False, out_html: bool = False,
+                                           out_dataset: bool = True,
```

Comment:
out_dataset is not defined in the method documentation. Also, out_dataset mirrors the same type you create within the method. I'm not sure if that could cause issues if you later try to use out_dataset, it won't be clear which one it should be.

Reply:
removed it. It is not needed and more and it was confusing...